vendor/init.te

```diff
@@ -6,17 +6,27 @@ allow init configfs:lnk_file { create unlink };
 allow init { mnt_vendor_file vendor_firmware_file adsprpcd_file }:dir mounton;
 allow init vendor_firmware_file:filesystem { mount relabelfrom };
 
+# Mount /vendor/dsp from fstab via init
+allow init adsprpcd_file:filesystem { relabelfrom relabelto mount };
```

Comment:
The decision to set contexts via fstab might controversial, only merge this in if we decide to follow that route.